target/linux/layerscape/patches-5.4/302-dts-0074-arm64-dts-fsl-ls1028a-Add-Felix-switch-port-DT-node.patch

```diff
@@ -58,7 +58,7 @@ Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
 +				ports {
 +					#address-cells = <1>;
 +					#size-cells = <0>;
-+
++				
```

Comment:
You still add whitespaces into an empty line here without any (apparant) reason ...